core/src/test/java/nl/nn/adapterframework/management/bus/endpoints/TestWebServices.java

```diff
@@ -178,6 +179,21 @@ public void getOpenApiSpec() throws Exception {
 		MatchUtils.assertJsonEquals("JSON Mismatch", expectedJson, result);
 	}
 
+	@Test
+	public void getConfigurationOpenApiSpec() throws Exception {
+		assertEquals(1, ApiServiceDispatcher.getInstance().findMatchingConfigsForUri(API_LISTENER_ENDPOINT).size());
+
+		MessageBuilder<String> request = createRequestMessage("NONE", BusTopic.WEBSERVICES, BusAction.DOWNLOAD);
+		request.setHeader("type", "openapi");
+		request.setHeader("configuration", getConfiguration().getName());
+		Message<?> response = callSyncGateway(request);
+
+		String result = response.getPayload().toString();
+		result = result.replaceFirst("auto-generated at .* for", "auto-generated at -timestamp- for");
```

Comment:
It is better to use the wildcard `.*?` instead of `.*`.

`.*` is "greedy" and will search the entire string to see if there's another match on " for", instead of matching on the first it finds.
So it is slower and potentially does an unwanted replacement.